gramps/plugins/webreport/person.py

```diff
@@ -585,9 +588,12 @@ def individualpage(self, report, title, person):
             if sect13 is not None:
                 individualdetail += sect13
 
-            # display ancestor tree
+            # display ancestor tree  
```

Comment:
Remove trailing white space from here.

Reply:
OK, doing that now.  I presume this, and other changes, are still on my fork i.e. papadeltasierra/buchheim.  Let me know if not.